ui/perfherder/compare/CompareTableView.jsx

```diff
@@ -29,6 +29,50 @@ import RevisionInformation from '../../shared/RevisionInformation';
 import CompareTableControls from './CompareTableControls';
 import NoiseTable from './NoiseTable';
 
+export const getPerfCompareCompareBaseURL = function getOldCompareWithBaseViewURL(
+  originalProject,
+  originalRevision,
+  newProject,
+  newRevision,
+  framework,
+) {
+  return `https://perf.compare/compare-results?baseRev=${originalRevision}&baseRepo=${originalProject}&newRev=${newRevision}&newRepo=${newProject}&framework=${framework}`;
+};
+
+export const getPerfCompareCompareBaseSubtestsURL = function getPerfCompareCompareBaseSubtestsURL(
+  originalProject,
+  originalRevision,
+  newProject,
+  newRevision,
+  framework,
+  originalSignature,
+  newSignature,
+) {
+  return `https://perf.compare/subtestsCompareWithBase?baseRev=${originalRevision}&baseRepo=${originalProject}&newRev=${newRevision}&newRepo=${newProject}&framework=${framework}&baseParentSignature=${originalSignature}&newParentSignature=${newSignature}`;
+};
+
+export const getPerfCompareCompareOvertimeURL = function getPerfCompareCompareOvertimeURL(
+  originalProject,
+  newProject,
+  newRevision,
+  framework,
+  timeRange,
+) {
+  return `https://perf.compare/compare-over-time-results?baseRepo=${originalProject}&selectedTimeRange=${timeRange}&newRev=${newRevision}&newRepo=${newProject}&framework=${framework}`;
+};
+
+export const getPerfCompareCompareOvertimeSubtestsURL = function getPerfCompareCompareOvertimeSubtestsURL(
+  originalProject,
+  newProject,
+  newRevision,
+  framework,
+  timeRange,
+  originalSignature,
+  newSignature,
+) {
+  return `https://perf.compare/subtestsCompareOverTime?baseRepo=${originalProject}&newRev=${newRevision}&newRepo=${newProject}&framework=${framework}&interval=${timeRange}&baseParentSignature=${originalSignature}&newParentSignature=${newSignature}`;
```

Comment:
Use `selectedTimeRange` instead of `interval` and let's wait until the change is deployed in perfcompare before merging this one.